skunkbooth/utils/frames.py

```diff
@@ -204,7 +204,7 @@ def __init__(self, screen: Any, filters: Any, data: Any = None) -> None:
             f[1] = temp
             logging.info(f"{f[0].name} button created")
             filters_layout.add_widget(temp)
-            filters_layout.add_widget(Label(f[0].description, align=">"), 1)
+            filters_layout.add_widget(Label(f"{f[0].description}   ]", align=">"), 1)
```

Comment:
`]` is looking put of place. I tried with unicode pipe symbol with one space and it looks much less intrusive.
```diff
-            filters_layout.add_widget(Label(f"{f[0].description}   ]", align=">"), 1)
+            filters_layout.add_widget(Label(f"{f[0].description} │", align=">"), 1)
```